Automattic/wp-calypso
# My Home: Add Store education card

#### Changes proposed in this Pull Request

* Add a new card component for the store for launched sites with the `sell` intent
* Must be run with D75890-code and the API sandboxed

**Visual demo**

<img width="1069" alt="Screen Shot 2022-03-02 at 3 53 36 PM" src="https://user-images.githubusercontent.com/2124984/156447905-d6b166fa-5f93-4e9d-a760-00644fdef2cd.png">

#### Testing instructions

* Switch to this PR, navigate to `/home` on a launched site that's created through the seller experience flow
* Apply D75890-code to your sandbox and sandbox `public-api.wordpress.com`
* You should see the new "How to set up an online store" card in the educational section of My Home.


Related to #61467
